core/domain/services/admin/registrations/list_table/csv_reports/Answers.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace EventEspresso\core\domain\services\admin\registrations\list_table\csv_reports;
+
+use EE_Error;
+use EEH_Export;
+use EEM_Answer;
+use EEM_Question;
+use EEM_State;
+
+/**
+ * Class Answers
+ * Add question / answer columns to the CSV row
+ *
+ * @author  Hossein Rafiei
+ * @package EventEspresso\core\domain\services\admin\registrations\list_table\csv_reports
+ */
+class Answers
```

Comment:
maybe rename to `AnswersCSV` to help differentiate this from any other similarly named entities